src/pe_reports/stylesheet.py

```diff
@@ -69,32 +69,22 @@ def text_style_ov_val(self, font):
 class Graph:
     """Simple class to call chart attributes."""
 
-    def bar(self, slide, chart):
-        """Medium bar chart."""
-        chart.font.size = Pt(10)
-        chart.font.rgb = (20, 200, 100)
+    def ch_font_attr(self, chart):
+        """Font attr set."""
+        self.sm = chart.font.size = Pt(10)
+        self.lg = chart.font.size = Pt(28)
+        self.blue = chart.font.rgb = (20, 200, 100)
+        self.white = chart.font.rgb = (255, 255, 255)
```

Comment:
These values should really be _static_ constants, as they are the same for all object instances.

Reply:
@jsf9k I created constants for "Font Attributes" that can be applied across all graphic components using "fonts". This approach can be expanded as we build out a full report with various attributes.

Resolved in 3ff00e0